hud.lua

```diff
@@ -25,11 +25,13 @@ if minetest.get_modpath("areas") then
 	pos_x = 0.95
 end
 
-local function generate_hud(player, current_owner, has_access)
-	-- green if access
-	local color = 0xFFFFFF
-	if has_access then
+local function generate_hud(player, current_owner, has_access, claimed)
+	-- green if access, red if no access but claimed, grey for unclaimed
```

Comment:
Unrelated change. Please move this to a separate PR.

( *The text for unclaimed areas (`"Area owner: `) could be better, such as `Area is not claimed`.* )

Reply:
Yeah I need to pull those changes out so I can make a separate PR. (But first let's get the MineClone support stuff going)